### ISSUE TITLE
Report payouts: positions amount are not zero after report payouts 

### ISSUE THREAD

**Issue by elena-zh** [gnosis/conditional-tokens-explorer#485]
The issue is related to #465 and #365

I have a condition with 3 positions. 
I split each position again. As a result, I get 9 positions. 
I resolve the condition
When I try to redeem one of the positions, I have a successful transaction, however, no tokens are returned to my wallet and the position balance is not zero. 

See the details on the screenshot. 
![image.png](https://images.zenhubusercontent.com/5f55ec970a2e890ec10f7778/5324f3e8-e135-48ee-89b8-a46cb94c24da)

Link to the condition ID: https://conditional-tokens-explorer.dev.gnosisdev.com/#/conditions/0x00f498c5a90432da7dfffe663032b970060e83b935664ac5b4a47c7010059795

**Expected Result**: position balance is zero, tokens are returned to the wallet

**Comment by elena-zh** [comment]
retest the issue after the #513 is implemented

**Comment by hpmaxi** [comment]
@elena-zh After #513, can you check that this only happens with "special positions", because with positions split from one condition and then split that position again with *another* condition seems to works just fine.

**Comment by elena-zh** [comment]
@hpmaxi , yes, sure. I do believe that these 'special' positions will not appear in the list of the 'reedemable' positions in #513. However, I'm not sure. Let see)

**Comment by elena-zh** [comment]
The issue is still reproducible after the new design implementation. =(
